README.md

```diff
@@ -82,6 +82,30 @@ ansible --inventory=hosts-file cool-servers \
         --become --ask-become-pass --user="ian.kilmister"
 ```
 
+To scan for indicator strings on AWS instances that are accessible via
+[SSM](https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html),
+the `ioc_scan_by_host.sh` shell script has been provided in the `extras`
+directory:
+
+```console
+$ ./extras/ioc_scan_by_host.sh my-ioc-strings.txt i-0123456789abcdef0
+
+IOC List is: "192.168.1[.]1 sketchy-site[.]org IOC_HASH_1 IOC_HASH_2"
```

Comment:
Just as a discussion point does mentioning hashes make sense here? The Python logic in this project generates hashes of files and checks those while this script searches for strings in log files. I don't want to conflate the functionality of the two components as the script added in this PR does no hash generation. Maybe something more like:

```suggestion
IOC List is: "192.168.1[.]1 sketchy-site[.]org IOC_STRING_1 IOC_STRING_2"
```

Reply:
That is an excellent point.  I will make that change shortly.